_posts/2020-07-06-whats-so-super-about-super-tdma.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+comments: true
+title: What's so "super" about Super-TDMA ? 
+date: 06/07/2020
+author: Prasad Anjangi
+categories: howto
+feature-img: "assets/img/stdma/stdma-unet.jpg"
+thumbnail: "assets/img/stdms/stdma-unet.jpg"
+tags: [howto, tdma, medium access control, long propagation delays, throughput, performance boost]
+---
+
+Needless to say that underwater acoustic (UWA) networks might play a key role in many areas including marine, offshore and subsea industries in the future. There have been tremendous and impressive technological advances in underwater acoustic communications and networking field. The one that caught my attention and brought about a clear difference in the novel approaches that are needed to deal with challenges in UWA networks was the "exploitation" of large propagation delays that exist in UWA networks. Do you know that it is possible to achieve as much as 50% higher network throughput with specific network geometries and protocols in UWA networks when compared to radio-frequency (RF) based terrestrial wireless networks? It is an interesting and surprising fact that has led to further exploration of such techniques in practical settings. In this blog, without delving too much into the nitty-gritty details, let's try and understand, where that advantage comes from, through an example technique (*Super-TDMA*). We will try and understand the key idea behind this technique  and let's see why it is "super".
```

Comment:
```suggestion
Underwater acoustic (UWA) networks will play a key role in many areas including marine, offshore and subsea industries in the future. There have been tremendous and impressive technological advances in underwater acoustic communications and networking field. The one that caught my attention and brought about a clear difference in the novel approaches that are needed to deal with challenges in UWA networks was the "exploitation" of large propagation delays that exist in UWA networks. 

Do you know that it is possible to achieve as much as 50% higher network throughput with specific network geometries and protocols in UWA networks when compared to radio-frequency (RF) based terrestrial wireless networks? 

It is an interesting and surprising fact that has led to further exploration of such techniques in practical settings. In this blog, let's try and understand, where that advantage comes from, through an example technique (*Super-TDMA*). We will try and understand the key idea behind this technique and let's see why it is "super".
```

"50% higher network throughput" <- would this be normalized throughput? Without the context this sounds like we're 50% faster than a terrestrial network.

Reply:
Yes you are right. Modified the statement now to reflect that correctly.